### ISSUE TITLE
Hardcoded Sonic.exe-ish notes

### ISSUE THREAD

**Issue by KalashniPepsi** [ShadowMario/FNF-PsychEngine#10736]
### Describe your problem here.

So im currently trying to hardcode a note that’s similar to something like the static notes in Vs. Sonic.exe in how the penalty/effect is when you miss the note and you need to prevent the penalty/effect by hitting the note. I’ve gotten the notes’ skin on but I’m unsure on how to code the penalty in. Psych engine only has the hurtnote which gives you the penalty when you do hit it so I can’t reuse that, anyone got any ideas?

tl;dr How do i set a note penalty to the miss instead of the hit?
(I swear to god don’t tell me to just use LUA)

### Are you modding a build from source or with Lua?

Source

### What is your build target?

Windows x64

### Did you edit anything in this build? If so, mention or summarize your changes.

A lot of the playstate and some of note.hx and chartingstate.hx

**Comment by MemeHoovy** [comment]
https://github.com/DuskieWhy/exe-2.5-code

**Comment by KalashniPepsi** [comment]
> https://github.com/DuskieWhy/exe-2.5-code use code from there

I would but the static notes don’t work there, that’s actually one of the first things I checked and I even tried fixing it for fun at one point. Everything seems in place, can’t figure out why they don’t work.

**Comment by MemeHoovy** [comment]
> > https://github.com/DuskieWhy/exe-2.5-code use code from there
> 
> I would but the static notes don’t work there, that’s actually one of the first things I checked and I even tried fixing it for fun at one point. Everything seems in place, can’t figure out why they don’t work.

I have no idea why they don't work either

**Comment by ShadowMario** [comment]
the source code from exe is public just go there instead of making an issue or something, idfk
